treeherder/webapp/api/jobs.py

```diff
@@ -88,10 +89,15 @@ def list(self, request, project, jm):
         - count (10)
         - return_type (dict)
         """
+        MAX_JOBS_COUNT = 2000
         filter = UrlQueryFilter(request.query_params)
 
         offset = int(filter.pop("offset", 0))
-        count = min(int(filter.pop("count", 10)), 2000)
+        count = int(filter.pop("count", 10))
+
+        if count > MAX_JOBS_COUNT:
+            return Response({"error": "Specified count exceeds API MAX_JOBS_COUNT value: " + str(MAX_JOBS_COUNT)}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
concatenating values to strings is done with ``format()``, and it's best to wrap to keep things as close to 80 chars wide as possible like this:

    return Response(
        {"error": "Specified count exceeds API MAX_JOBS_COUNT value: {}".format(MAX_JOBS_COUNT)},
        status=status.HTTP_400_BAD_REQUEST
    )